php/media/class-gallery.php

```diff
@@ -398,6 +396,39 @@ protected function register_settings() {
 		$media_settings->add_setting( $this->settings );
 	}
 
+	/**
+	 * Inits the cloudinary gallery using block attributes.
+	 *
+	 * @param string $content The post content.
+	 * @param array  $block   Block data.
+	 *
+	 * @return string
+	 */
+	public function prepare_block_render( $content, $block ) {
+		if ( 'cloudinary/gallery' !== $block['blockName'] ) {
+			return $content;
+		}
+
+		$attributes                = Utils::expand_dot_notation( $block['attrs'], '_' );
+		$attributes                = array_merge( self::$default_config, $attributes );
+		$attributes['mediaAssets'] = $attributes['selectedImages'];
+		$attributes['cloudName']   = $this->media->plugin->components['connect']->get_cloud_name();
+		unset( $attributes['selectedImages'], $attributes['customSettings'] );
+
+		ob_start();
+		?>
+<script>
```

Comment:
The indentation here appears to be different from the rest of the method -- is that intentional?